app/views/headerFooterTemplate/Layout.scala.html

```diff
@@ -59,11 +59,14 @@
 
 @footerBlock = {
     @hmrcStandardFooter()
-    <script @{CSPNonce.attr} src='@controllers.routes.Assets.versioned("lib/govuk-frontend/govuk/all.js")'></script>
-    <script @{CSPNonce.attr} src='@controllers.routes.Assets.versioned("lib/hmrc-frontend/hmrc/all.js")'></script>
-    <script @{CSPNonce.attr} src='@controllers.routes.Assets.versioned("javascripts/application.js")'></script>
-    <script @{CSPNonce.attr}>window.GOVUKFrontend.initAll();</script>
-    <script @{CSPNonce.attr}>window.HMRCFrontend.initAll();</script>
+    <script type="module" @{CSPNonce.attr} src='@controllers.routes.Assets.versioned("lib/govuk-frontend/dist/govuk/all.js")'></script>
```

Comment:
This needs the path changing to '/all.bundle.js' or the file will not load.

Reply:
Updated.